topics/tailwind/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss
+created_by: Adam Wathan, Jonathan Reinink, David Hemphill, Steve Schoger
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss
+released: October 2017
+short_description: Tailwind is a Utility-First CSS framework.
+topic: tailwind
+url: https://tailwindcss.com/
+---
+A Utility-First CSS Framework for Rapid UI Development
```

Comment:
Could you lowercase this and add a period?

Can this be expanded? It should give more information than what the `short_description` does. For example: why would I choose Tailwind over Bootstrap/Bulma/Materialize? Is Tailwind particularly smaller than other CSS frameworks? Does it support mobile? Does it support older browsers? Etc.